extensions/wikia/SASS/SassUtil.php

```diff
@@ -7,7 +7,7 @@ class SassUtil {
 
 	const DEFAULT_OASIS_THEME = 'oasis';
 	const HEX_REG_EXP = '/#([a-f0-9]{3,6})/i';
-	const THEME_DESIGNER_COLOR_KEYS = array('color-body', 'color-body-middle', 'color-page', 'color-buttons', 'color-links', 'color-header');
+	const THEME_DESIGNER_COLOR_KEYS = array('color-body', 'color-body-middle', 'color-page', 'color-buttons', 'color-links', 'color-local-nav', 'color-header');
```

Comment:
I won't be changing color-header now, I want to focus on things that are connected with our goal right now aka It's not trivial to change this name (e.g. I'd have to change how we store this data in WikiFactory var)